dataland-frontend/src/utils/smoothScroll.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * Smooth scroling
+ *
+ * @param target - target element
+ */
+export function smoothScroll(target: string): void {
+  const targetElement = document.querySelector(target) as HTMLElement;
+  const targetPosition = targetElement.getBoundingClientRect().top + window.scrollY - 100;
+  const startPosition = window.scrollY;
+  const distance = targetPosition - startPosition;
+  const duration = 300;
+  let startTime = null as unknown as number;
+
+  const animation = (currentTime: number): void => {
```

Comment:
Can you please explain me where the actual value for "currentTime" is passed to this function?